Lecture_1/task_2/src/main/java/task_2.java

```diff
@@ -0,0 +1,39 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+class task_2 {
+
+    public static void main(String[] args) throws IOException {
+
+                System.out.println("Введите номер дня недели:");
```

Comment:
it is better to move all `reading` code into a private method  - and catch all necessary exception there